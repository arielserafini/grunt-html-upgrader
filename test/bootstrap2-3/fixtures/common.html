<img src="something"/>

<ul class="dropdown-menu">
  <li class="nav-header">header</li>
</ul>

<%= render 'tango/templates' %>
