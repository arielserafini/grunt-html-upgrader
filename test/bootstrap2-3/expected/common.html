<img class="img-responsive" src="something"/>

<ul class="dropdown-menu">
  <li class="dropdown-header">header</li>
</ul>
